index.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+    <header class="container">
+        <div class="logo">My Logo</div>
+
+        <nav class="menu-icon">
+            <img src="./images/ic_menu.jpg" alt="">
+        </nav>
```

Comment:
- [ ]  it's very important to stick with the Figma design. As you see the menu icon is getting out of the layout for some reason, please try to figure out the reason and fix it.

![image](https://user-images.githubusercontent.com/34779928/191850405-35852468-f81b-4fe7-bb27-3f8b99c299bd.png)